Install fails with eio_req error

On Mac OS X Mountain Lion, I got the following error on install:

```
> node-gyp clean rebuild

  CXX(target) Release/obj.target/opencv/src/init.o
In file included from ../src/init.cc:4:
../src/CascadeClassifierWrap.h:16:35: error: unknown type name 'eio_req'
        static void EIO_DetectMultiScale(eio_req *req);
                                         ^
../src/CascadeClassifierWrap.h:17:39: error: unknown type name 'eio_req'
        static int EIO_AfterDetectMultiScale(eio_req *req);
                                             ^
2 errors generated.
make: *** [Release/obj.target/opencv/src/init.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:789:12)
gyp ERR! System Darwin 12.5.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "clean" "rebuild"
gyp ERR! cwd node_modules/faced/node_modules/opencv
gyp ERR! node -v v0.10.8
gyp ERR! node-gyp -v v0.9.5
gyp ERR! not ok 
npm ERR! weird error 1
npm ERR! not ok code 0
```

After determining the error was in the version of the opencv library included as a dependency with Faced, I created the below pull request to bump the version. It installs correctly and works for the example. I haven't tested it more extensively than that.

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
     "example": "examples"
   },
   "dependencies": {
-    "opencv": "~0.0.9",
+    "opencv": "~0.3.1",
     "underscore": "~1.4.4"
   },
   "devDependencies": {},

